internal/app/backend/backend.go

```diff
@@ -23,30 +23,23 @@ import (
 	"open-match.dev/open-match/pkg/pb"
 )
 
-var (
-	backendLogger = logrus.WithFields(logrus.Fields{
```

Comment:
I remember you setting this up earlier with an intent to have one per file - now we are switching to a logger per package? What is the intent and generally the recommendation for Open Match components going forward?

Reply:
Originally the intent was to mirror what Open Match had before for consistency. After looking at how the server runs it's not very helpful to distinguish exactly which file the log statement comes from and there's options to provide better feedback here. Also this is separation will be further diminished when we fix up the error handling.